docs/installation/installation-using-docker-macos.md

```diff
@@ -27,12 +27,38 @@ There are two domains each for different language in default installation. First
 sudo ifconfig lo0 alias 127.0.0.2 up
 ```
 
-### 2. Create docker-compose.yml file
+### 2. Create docker-compose.yml and docker-sync.yml
 Create `docker-compose.yml` from template [`docker-compose-mac.yml.dist`](../../project-base/docker/conf/docker-compose-mac.yml.dist).
 ```
 cp docker/conf/docker-compose-mac.yml.dist docker-compose.yml
 ```
 
+Create `docker-sync.yml` from template [`docker-sync.yml.dist`](../../project-base/docker/conf/docker-sync.yml.dist).
+
+### 2.1 Set the UID and GID to allow file access in mounted volumes
+Because we want both the user in host machine (you) and user running php-fpm in the container to access shared files, we need to make sure that they both have the same UID and GID.
+This can be achieved by build arguments `www_data_uid` and `www_data_gid` that should be set to the UID and GID as your own user in your `docker-compose.yml`.
```

Comment:
"...should be set to the **same** UID and GID..."

Reply:
still missing the "same" word here :slightly_smiling_face:

Reply:
it is mixed up somehow - the section `### 2.1` and the section `### 3`, check it out please